lib/oli_web/live/products/details_view.ex

```diff
@@ -145,4 +169,44 @@ defmodule OliWeb.Products.DetailsView do
         {:noreply, assign(socket, changeset: changeset)}
     end
   end
+
+  def handle_event("validate_image", _, socket) do
```

Comment:
I think we are using this event in the component but we are not passing it as an explicit prop, am I right?

Reply:
Yes, `live_file_input` (used in Surface `LiveFileInput`) requires to be in a form with `phx-submit` and `phx-change` events. Even if you do nothing on change.